server/src/server/api/middleware/requestLoggerMiddleware.ts

```diff
@@ -6,7 +6,8 @@ export const RequestLogger = (
   next: NextFunction
 ) => {
 
-  console.info(`${request.method} ${request.path}`);
+  // 4/2/2020, 10:39:06 AM: [1347a8a782381] Request GET - /
+  console.info(`${new Date().toLocaleString()}: [${(request as any).id}] Request ${request.method} - ${request.path}`);
   
```

Comment:
Lets output this in json. and use a UTC timestamp

Reply:
where is the correlationId?

Reply:
`[${(request as any).id}]` is the correlationId

Reply:
Changed output to JSON and now using UTC - 0f5d7ca